### ISSUE TITLE
Error in config.yaml nginx configuration section 

### ISSUE THREAD

**Issue by extead** [puphpet/puphpet#1896]
Hello. I'am create archive with Puphpet today. 

Vagrant provision tell me:

==> default: Error: "off" is not a boolean.  It looks to be a String at /tmp/vagrant-puppet/modules-e7f1356ed6ef4982a79cb9013062f1eb/nginx/manifests/resource/location.pp:198 on node site.dev
==> default: Error: "off" is not a boolean.  It looks to be a String at /tmp/vagrant-puppet/modules-e7f1356ed6ef4982a79cb9013062f1eb/nginx/manifests/resource/location.pp:198 on node site.dev
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

My config.yaml (nginx section):
```
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
    upstreams: {  }
    vhosts:
        nxv_oygvffewi132:
            server_name: site.dev
            server_aliases:
                - www.site.dev
            www_root: /var/www/site
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_efqlpni00rtp:
                    location: /
                    autoindex: "off"
                    internal: "off"
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                nxvl_ofep5ueh9789:
                    location: '~ \.php$'
                    autoindex: "off"
                    internal: "off"
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
    proxies: {  }
```

I replaced all "off" with false and it provisioned correctly. I think this is a bag.

**Comment by emielmolenaar** [comment]
Dupe of #1882, #1881, #1869.

**Comment by jtreminio** [comment]
Fixed by 36878e0
